packages/flutter/lib/src/material/icon_button.dart

```diff
@@ -92,6 +97,17 @@ const double _kMinButtonSize = kMinInteractiveDimension;
 /// ** See code in examples/api/lib/material/icon_button/icon_button.1.dart **
 /// {@end-tool}
 ///
+/// Material Design 3 introduced new types (standard and contained) of [IconButton]s.
+/// The default [IconButton] is the standard type, and contained icon buttons can be produced
+/// by configuring the [IconButton] widget's properties.
```

Comment:
```suggestion
/// by providing [style].
```

If defaults are added, mention them here "as a starting point ...standard.copyWith(...)"